easybuild/easyconfigs/p/PyBERT/PyBERT-1.7.9_2to3.patch

```diff
@@ -0,0 +1,415 @@
+# Python 2to3
```

Comment:
@hajgato how sure are you this is enough to make `PyBERT` Py3 compatible? seems like a long shot to me...